docs/synthetics/http-monitor.md

```diff
@@ -41,15 +43,29 @@ If you disable this option, we **won’t save the response body** hence it **wil
 
 ## Authentication
 
-HTTP monitors support header-based authentication. Custom HTTP request headers can be specified when creating a monitor. To use Basic authentication, add a custom request header according to the following specifications:
+You can **dynamically fetch a token** for each run with token support and pass that token in your API requests. 
+
+When creating an HTTP monitor, navigate to the Authentication tab and select **Bearer/Access Token** authentication type.
+
+![Access Token Authentication](../images/synthetics/authentication-token.png)
+
+Enter the access token url and the source field to get the token from response. Enter body parameters to fetch the token such as grant-type, username and password.
+
+Before Sematext Cloud calls the endpoint an access token will be fetched based on the parameters entered within this tab and it will be passed to the API request.
+
+You also have the option to select **Basic Authentication** type within the Authentication tab and pass username and password to connect your **protected APIs**. 
+
+![Basic Authentication](../images/synthetics/authentication-basic.png)
+
+HTTP monitors also support header-based authentication. Custom HTTP request headers can be specified when creating a monitor. To use Basic authentication, add a custom request header according to the following specifications:
 
 | Name | Value | Description |
 | --- | --- | --- |
 | `Authorization` | `Basic XYZ` | **XYZ** is the **Base64** encoding of the **username:password** string. |
 
 After **Base64** encoding the string `username:password` above, would become `dXNlcm5hbWU6cGFzc3dvcmQ=`. This is then prepended with `Basic` to get the final `Authorization` header value of `Basic dXNlcm5hbWU6cGFzc3dvcmQ=`.
```

Comment:
After **Base64** encoding the string `username:password` above, would become `dXNlcm5hbWU6cGFzc3dvcmQ=`.
=> 
After **Base64** encoding, the string `username:password` above would become `dXNlcm5hbWU6cGFzc3dvcmQ=`.